mne/viz/_brain/_brain.py

```diff
@@ -2316,6 +2317,43 @@ def add_label(self, label, color=None, alpha=1, scalar_thresh=None,
             self._labels[hemi].append(label)
         self._renderer._update()
 
+    @fill_doc
+    def add_head(self, dense=True, color=None, alpha=0.5):
+        """Add a mesh to render the outer head surface.
+
+        Parameters
+        ----------
+        dense : bool
+            Whether to plot the dense head (``seghead``) or the less dense head
+            (``head``).
+        color : matplotlib-style color | None
+            A list of anything matplotlib accepts: string, RGB, hex, etc.
+            (default: "gray").
+        alpha : float in [0, 1]
+            Alpha level to control opacity.
```

Comment:
I'm a little surprised this passed our docstring tests.  I think I would have done
```suggestion
        alpha : float
            Opacity of the head surface. Must be between 0 and 1 (inclusive).
            Default is 0.5.
```
because `float in [0, 1]` reads like Python code, confusingly suggesting this param can take on one of 2 values (`0` or `1`)

Reply:
Eric wrote that :)